demo/tcf/secondLayer/demo/index.js

```diff
@@ -11,14 +13,46 @@ const TcfSecondLayerDemo = () => {
     console.log('purposes', purpose)
     console.log('vendors', vendor)
   }
+  const uiVisible = ({visible}) => {
+    console.log('uiVisible called with visible to', visible)
+  }
+  const handleGoBack = () => {
+    console.log('go back button pressed')
+  }
+  const [isMobile, setIsMobile] = useState(false)
 
   return (
-    <TcfSecondLayer
-      isOpen
-      loadUserConsent={loadUserConsent}
-      saveUserConsent={saveUserConsent}
-      getVendorList={getVendorList}
-    />
+    <>
+      <button
+        onClick={() => setIsMobile(!isMobile)}
+        style={{height: '25px', width: '100%', backgroundColor: '#2b91c1'}}
+      >
+        {isMobile ? 'Click to change to Desktop' : 'Click to change to Mobile'}
+      </button>
+      {isMobile && (
+        <TcfSecondLayer
+          isOpen
+          logo="https://frtassets.fotocasa.es/img/fotocasa_logo.svg"
+          loadUserConsent={loadUserConsent}
+          saveUserConsent={saveUserConsent}
+          getVendorList={getVendorList}
+          uiVisible={uiVisible}
+          onGoBack={handleGoBack}
+          isMobile
+        />
+      )}
+      {!isMobile && (
+        <TcfSecondLayer
+          isOpen
+          logo="https://frtassets.fotocasa.es/img/fotocasa_logo.svg"
+          loadUserConsent={loadUserConsent}
+          saveUserConsent={saveUserConsent}
+          getVendorList={getVendorList}
+          uiVisible={uiVisible}
+          onGoBack={handleGoBack}
+        />
+      )}
```

Comment:
Why are you repeating this? 
Try <TcfSecondLayer ... isMobile={isMobile} />

Reply:
thanks